From f59fdbdfc3f22c1aaea59c77cc082b63d07daac4 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:10:15 -0400
Subject: [usb] cp210x: add support for Corsair usb dongle

Message-id: <1410887463-20674-544-git-send-email-dzickus@redhat.com>
Patchwork-id: 95664
O-Subject: [RHEL7 PATCH 543/591] USB: cp210x: add support for Corsair usb dongle
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit b9326057a3d8447f5d2e74a7b521ccf21add2ec0
Author: Andras Kovacs <andras@sth.sze.hu>
Date:   Fri Jun 27 14:50:11 2014 +0200

    USB: cp210x: add support for Corsair usb dongle

    Corsair USB Dongles are shipped with Corsair AXi series PSUs.
    These are cp210x serial usb devices, so make driver detect these.
    I have a program, that can get information from these PSUs.

    Tested with 2 different dongles shipped with Corsair AX860i and
    AX1200i units.

    Signed-off-by: Andras Kovacs <andras@sth.sze.hu>
    Cc: <stable@vger.kernel.org>
    Signed-off-by: Johan Hovold <johan@kernel.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/cp210x.c b/drivers/usb/serial/cp210x.c
index 762e4a5..330df5c 100644
--- a/drivers/usb/serial/cp210x.c
+++ b/drivers/usb/serial/cp210x.c
@@ -153,6 +153,7 @@ static const struct usb_device_id id_table[] = {
  { USB_DEVICE(0x1843, 0x0200) }, /* Vaisala USB Instrument Cable */
  { USB_DEVICE(0x18EF, 0xE00F) }, /* ELV USB-I2C-Interface */
  { USB_DEVICE(0x1ADB, 0x0001) }, /* Schweitzer Engineering C662 Cable */
+ { USB_DEVICE(0x1B1C, 0x1C00) }, /* Corsair USB Dongle */
  { USB_DEVICE(0x1BE3, 0x07A6) }, /* WAGO 750-923 USB Service Cable */
  { USB_DEVICE(0x1E29, 0x0102) }, /* Festo CPX-USB */
  { USB_DEVICE(0x1E29, 0x0501) }, /* Festo CMSP */
-- 
1.7.1